src/ray/core_worker/actor_manager.cc

```diff
@@ -88,9 +89,12 @@ bool ActorManager::AddActorHandle(std::unique_ptr<ActorHandle> actor_handle,
         std::bind(&ActorManager::HandleActorStateNotification, this,
                   std::placeholders::_1, std::placeholders::_2);
     RAY_CHECK_OK(gcs_client_->Actors().AsyncSubscribe(
-        actor_id, actor_notification_callback, nullptr));
-  } else {
-    RAY_LOG(ERROR) << "Actor handle already exists " << actor_id.Hex();
```

Comment:
Should this log be kept?

Reply:
We don't need this and it would confuse users.